src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/JsonConverter.FromJson.ReadOnlySequence.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+
+namespace System.Text.Json.Serialization
+{
+    public static partial class JsonConverter
+    {
+        public static T FromJson<T>(this ReadOnlySequence<byte> json, JsonConverterSettings settings = null)
```

Comment:
Pass these by `in` I wonder how we feel about those in public APIs

Reply:
We have it on the `Utf8JsonReader`.

Reply:
OK I'll add `in` to the ReadOnlySequence APIs